qemu.go

```diff
@@ -87,7 +87,7 @@ var supportedQemuMachines = []ciaoQemu.Machine{
 	},
 	{
 		Type:         QemuQ35,
-		Acceleration: "kvm,kernel_irqchip,nvdimm,nosmm,nosmbus,nosata,nopit,nofw",
```

Comment:
We need to make sure to add the proper things to the runtime when vendoring this change. Indeed, without the extra accelerators params, q35 machine type won't work.